src/js/themes/hpe.js

```diff
@@ -266,6 +280,19 @@ export const hpe = deepFreeze({
         },
         color: undefined,
       },
+      extend: ({ colorValue, primary, theme }) => `
```

Comment:
Is there a reason we cannot use `theme.button.hover.primary.background` for this, with `dark` and `light` sub-keys?

Reply:
so the reason I have this is to keep backward compatibility until `colored` button is discussed and researched further. Since I have the above `button.hover.primary.background` set to the dark green above this would also change the `colored` button since we pass primary color here: 
```
      primary: {
        // Designs call for using text-strong for the text color.
        // Keeping the label color the same on hover to maintain readablility.
        // color: hpe.global.colors['text-strong'].light,
        background: {
          dark: 'validation-ok',
          // Designs call for using a lower opacity of exisiting green!
          // instead of creating a new color name for this single usecase.
          light: {
            color: 'green!',
            opacity: 0.75,
          },
        },
      },
```
So in order to keep things the way they are I added this extend for `Primary && colorValue` are used
Im thinking we can enhance grommet to also have something such as `theme.button.hover.color.primary.baackground` then we can set a dark and light in there. I was holding off on this until the research for hover on colored buttons was complete.